package.json

```diff
@@ -38,7 +38,10 @@
     "generate:readme:toc": "markdown-toc -i \"README.md\"",
     "generate:assets": "npm run build && npm run generate:readme:toc",
     "bump:version": "npm --no-git-tag-version --allow-same-version version $VERSION",
-    "prepublishOnly": "npm run generate:assets"
+    "prepublishOnly": "npm run generate:assets",
+    "changeset": "changeset",
+    "version-packages": "changeset version",
+    "publish-packages": "turbo run build && changeset publish"
```

Comment:
instead of adding those to package.json did you consider just using `npx` and package directly in release config?

in past with semantic-release we resigned from using npm commands if they were not needed during regular development. So semantic-release we used directly like `npx semantic-release@19.0.4` so we had less things to maintain, it was easier to reuse across repos, and dependabot was not bugging us with new versions bump

Reply:
ok now added.